collector/collector_test.go

```diff
@@ -19,7 +19,7 @@ import (
 	"testing"
 	"time"
 
-	"github.com/nats-io/go-nats-streaming"
+	stan "github.com/nats-io/go-nats-streaming"
```

Comment:
we should update this to `github.com/nats-io/stan.go` soon probably... can do after this PR if it does not affect the build

Reply:
dunno why it changed the import 🤔